student.rb

```diff
@@ -0,0 +1,120 @@
+require 'csv'
+require_relative 'calculator'
+class Student
+
+	attr_accessor :name, :roll_number, :bengali, :english, :maths, :computer, :total, :percentage, :percentile
+	def initialize name, roll, bengali,english,maths,computer,t,per
+		@name=name
+		@roll_number = roll
+		@bengali = bengali
+		@english = english
+		@maths = maths
+		@computer = computer
+		@total=t
+		@percentage=per
+		@percentile=0
+	end
+
+	def calculate_total
+		@total=Calculator.new.add(@total,@bengali)
+		@total=Calculator.new.add(@total,@english)
+		@total=Calculator.new.add(@total,@maths)
+		@total=Calculator.new.add(@total,@computer)
```

Comment:
Why are you creating multiple objects? You can create one object and call add method.